server/devpi_server/model.py

```diff
@@ -311,6 +317,8 @@ def create_stage(self, index, type="stage", **kwargs):
         if index in self.get().get("indexes", {}):
             raise InvalidIndex("indexname '%s' already exists" % index)
         stage = self._getstage(index, type, {"type": type})
+        if isinstance(stage.customizer, UnknownCustomizer):
+            raise InvalidIndexconfig("unknown index type %r" % type)
```

Comment:
I see no tests for this ``InvalidIndexconfig``.

Reply:
Covered by test_create_with_invalid_type